agent/consul/leader.go

```diff
@@ -1270,6 +1270,16 @@ func (s *Server) handleAliveMember(member serf.Member) error {
 			ID:      structs.ConsulServiceID,
 			Service: structs.ConsulServiceName,
 			Port:    parts.Port,
+			Weights: &structs.Weights{
+				Passing: 1,
+				Warning: 1,
+			},
+			Meta: map[string]string{
+				"consul_voter":  strconv.FormatBool(!parts.NonVoter),
+				"raft_version":  strconv.Itoa(parts.RaftVersion),
+				"serf_protocol": fmt.Sprintf("%v (%v .. %v)", member.ProtocolCur, member.ProtocolMin, member.ProtocolMax),
+				"version":       parts.Build.String(),
```

Comment:
This could maybe be `"consul_version"` (we could add this meta to agents as well in the future and this could conflict with user defined meta)

Reply:
@Aestek version sounds good as it is namespaced by consul service name (and AFAIK, it is not possible to add other meta to consul service)